app/presenters/pagination_presenter.rb

```diff
@@ -0,0 +1,16 @@
+class PaginationPresenter
+  include Kaminari::Helpers::HelperMethods
+  attr_reader :page, :total_items, :total_pages
+
+    def initialize(page: 1, total_items:)
+      @items_per_page = 5
```

Comment:
I felt as I was using the app that 5 results per page was too few, but that's just personal preference.

Reply:
5 was a fairly arbitrary amount of items per page - I'll increase this.

Reply:
Updated to 15 per page.